chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/endpoints/dto/ChaosMonkeyDisabledDto.java

```diff
@@ -1,10 +1,52 @@
 package de.codecentric.spring.boot.chaos.monkey.endpoints.dto;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.time.Duration;
 import java.time.ZonedDateTime;
 import lombok.Data;
 
 @Data
 public class ChaosMonkeyDisabledDto {
   private final String status = "Chaos Monkey is disabled";
-  private ZonedDateTime disabledAt = ZonedDateTime.now();
+  private final ZonedDateTime disabledAt = ZonedDateTime.now();
+  @JsonIgnore private Duration timeActive;
+  @JsonIgnore private Duration timeInActive;
```

Comment:
nitpicking here 🤓 If spotless allows it, put the `@JsonIgnore` above the field:
eg.: 
```java
@JsonIgnore
private Duration timeActive;
```